hrjobcontract/js/src/job-contract/directives/loader.js

```diff
@@ -1,56 +1,57 @@
+/* eslint-env amd */
+
 define([
-    'job-contract/directives/directives'
+  'job-contract/directives/directives'
 ], function (directives) {
-    'use strict';
-
-    directives.directive('hrjcLoader',['$rootScope','$log', function ($rootScope, $log) {
-        $log.debug('Directive: hrjcLoader');
+  'use strict';
 
-        return {
-            link: function ($scope, el, attrs) {
-                var loader = document.createElement('div'),
-                    loaderSet = false,
-                    positionSet = false;
+  directives.directive('hrjcLoader', ['$rootScope', '$log', function ($rootScope, $log) {
+    $log.debug('Directive: hrjcLoader');
 
-                loader.className = 'hrjc-loader';
+    return {
+      link: function ($scope, el, attrs) {
+        var loader = document.createElement('div');
+        var loaderSet = false;
```

Comment:
should be `isLoaderSet`, because otherwise it can be read as `a loader set` or `a set of loaders` where "set" is a noun, not a verb. Same for `positionSet`.